src/TLuaInterpreter.cpp

```diff
@@ -11953,6 +11953,19 @@ double TLuaInterpreter::condenseMapLoad()
     return loadTime;
 }
 
+int TLuaInterpreter::getAvailableFonts(lua_State* L)
+{
+    auto fontList = mudlet::self()->getAvailableFonts();
+
+    lua_newtable(L);
+    for (auto& font : fontList) {
+        lua_pushstring(L, font.toUtf8().constData());
+        lua_pushboolean(L, true);
```

Comment:
If we logged the (positive int) id when we loaded each font we could perhaps use them here (and potentially allows for removal in the future even though you/we are not envisioning it).

Reply:
Oh, I've just spotted how you propose to enable that `if getAvailableFonts()["Ubuntu Mono"] then ...` to work, but my suggestion requires there be a lua command to return whether a table ***value*** is present in a table and there is not a quick and easy methodology for that, *is* there...? :man_shrugging:

Reply:
That's correct, this is for easy and convenient font presence checking.

Reply:
This is working with the font names - which may not have a direct correspondence with the font file names.

Okay, I guess, if it comes to removing things in the future, I can always cook-up a `getLoadedFontFiles()` returning the `key: (int)FontID / value: (string)PathFileName` table I was hinting at - and let the user work out the relationship between file names and font names - if they can - because it ain't simple is it?